pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -511,7 +511,7 @@ let
         DRM_AMD_DC_DCN = lib.mkIf (with stdenv.hostPlatform; isx86 || isPower64) (
           whenBetween "5.11" "6.4" yes
         );
-        DRM_AMD_DC_FP = whenAtLeast "6.4" yes;
+        DRM_AMD_DC_FP = lib.mkIf (!stdenv.cc.isClang) (whenAtLeast "6.4" yes);
```

Comment:
Is that still needed? I was curious about the failure and at least `pkgsLLVM.linux` seems to build when flipping the condition.

Reply:
Idk but without doing this change, I get an error saying that the option wasn't recognized.

Reply:
For which attribute-path specifically? I may just have missed a case.

Reply:
`pkgsLLVM.linux.configfile` at build time

Reply:
Would be good to understand why.

Reply:
Added a comment to explain why.